### PR TITLE
add task solution MR

### PR DESCRIPTION
**Pull request by RuslanMelnykFE**
- [DEMO LINK](https://RuslanMelnykFE.github.io/layout_stars/)
- [TEST REPORT LINK](https://RuslanMelnykFE.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
...
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars stars--1">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
```
Comment (nazarmatsevych) [inline src/index.html:25]:
```suggestion
      <ul class="stars stars--0">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
      
      <ul class="stars stars--1">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
```

Add blank line between multiple sibling blocks of html

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (nazarmatsevych) [inline src/style.css:3]: Don't reset paddings and margins in * selector, reset them for tags you are using

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Code looks good but your links are not working, try to run npm run deploy command again

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**